addons/mail/models/ir_ui_view.py

```diff
@@ -6,3 +6,10 @@ class View(models.Model):
     _inherit = 'ir.ui.view'
 
     type = fields.Selection(selection_add=[('activity', 'Activity')])
+
+    def _postprocess_tag_field(self, node, name_manager, node_info):
```

Comment:
Both.

At first, it was to avoid injecting the embedded views for the three x2many fields `message_ids`, `follower_ids`, `activity_ids`, as well as their `fields_get` description, in the result of every `load_views`, as it's unused by the web client because this is processed in a complete custom way. The only thing checked by the web client it's the presence of these fields, and no more.

Then, it's an optimization as well, as all the things the post-processing was adding (the modifiers, ...) was completely left unused by the web client.

But you are right, it deserves a comment in the method.